### ISSUE TITLE
Custom format for %H with precision misses author

### ISSUE THREAD

**Issue by aaccomazzi** [adsabs/export_service#94]
The output for `%H` applied to multi-authored papers misses the next-to-last author name when a precision is specified.  For example, consider the paper `2019AAS...23338108A` and the output generated from the following formats:
```
%2H -- Accomazzi Henneken (should be "Accomazzi Kurtz")
%3H -- Accomazzi Kurtz and Grant (should be "Accomazzi Kurtz Henneken Grant")
```

**Comment by golnazads** [comment]
So sorry. This is real bad. thank you so much for discovering it. I think I know what is happening, I split and then reattached, apparently I miss calculate the index. thank you.

**Comment by golnazads** [comment]
No it is actually missing the second number, that is why no one had discovered it yet, lol. And was happening for all of the author specifiers. Fixed it. thank you so much.

**Comment by golnazads** [comment]
fixed.

**Comment by golnazads** [comment]
fixed in v2.0.39.
